LibreNMS/OS/AirosAfLtu.php

```diff
@@ -37,7 +37,10 @@ public function discoverWirelessFrequency()
      */
     public function discoverWirelessDistance()
     {
-        $oid = '.1.3.6.1.4.1.41112.1.10.1.4.1.23.24.232.41.30.56.214'; //UBNT-AFLTU-MIB::afLTUStaRemoteDistance.1
+        $oid = snmp_getnext($this->getDevice(), '.1.3.6.1.4.1.41112.1.10.1.4.1.23', '-OnQ'); //UBNT-AFLTU-MIB::afLTUStaRemoteDistance
+        $oid = explode('=', $oid, 2);
+        $oid = trim($oid[0]);
+
         return array(
             new WirelessSensor('distance', $this->getDeviceId(), $oid, 'airos-af-ltu', 1, 'Distance', null, 1, 1000),
```

Comment:
You have the value too, so send that to the new WirelessSensor call instead of null